File: Module-CP/Assignments/assignment14/Pass_by_value/q9.c
#include<stdio.h>
typedef struct Complex{
    int real;
    int imag;
}Complex;
Complex storeDate();
void display(Complex);
int main(){
    Complex c1,c2;
    c1=storeDate();
    c2=storeDate();
    printf("Compex Num-1:");
    display(c1);
    printf("\nCompex Num-2:");
    display(c2);
}
Complex storeDate(){
    Complex c;
    printf("Enter real:");
    scanf("%d",&c.real);
    printf("Enter imag:");
    scanf("%d",&c.imag);

    return c;
}
void display(Complex c){
     printf("\n%d + %di",c.real,c.imag);
}

File: Module-Core Java/Assignments/C to Java/Assignment 14/Main/q9.java
import java.util.*;
class Complex{
    int real,imag;
}
public class q9{
    public static void main(String[] args) {
        Complex d1 = new Complex();
        Complex d2 = new Complex();

        Scanner sc = new Scanner(System.in);
        
        System.out.println("real");
        d1.real=sc.nextInt();
        System.out.println("imag");
        d1.imag=sc.nextInt();
        System.out.println("real");
        d2.real=sc.nextInt();
        System.out.println("imag");
        d2.imag=sc.nextInt();

          System.out.println(d1.real+ "+" +d1.imag+"i");
          System.out.println(d2.real+ "+" +d1.imag+"i");
    }
}